content/millennium/r4/individuals/related-person.md

```diff
@@ -105,30 +105,6 @@ Notes:
 
 <%= disclaimer %>
 
-#### Patient Authorization Request
```

Comment:
The new extensions need to be mentioned above in the Custom Extensions section and where "The following fields are returned if valued:"

Reply:
added here https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR23-R24 
and here
https://github.com/cerner/fhir.cerner.com/pull/624/commits/f8e3e624c57363c311b7e4577162e926522db9c8#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR50-R51